src/plugins/exploitmon/exploitmon.h

```diff
@@ -0,0 +1,132 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
```

Comment:
Please re-paste the license from the latest LICENSE file, there was a typo that got fixed that's not included in here

Reply:
Done